symplyphysics/laws/dynamics/mechanical_work_during_rotation.py

```diff
@@ -88,11 +88,14 @@
 
 angle_start = Symbol("angle_start", angle_type)
 angle_end = Symbol("angle_end", angle_type)
+# Infinitesimal work = dTau(angle) * dAngle
```

Comment:
But since $`W = \tau \theta`$, then $`dW = d(\tau \theta) = \theta d\tau + \tau d\theta`$, not $`d\tau d\theta`$

Reply:
Or you mean that we found infinitesimal work earlier, but in that case we just have $`dW = \tau d\theta`$

Reply:
Yes, dW = Tau * dTheta

Reply:
You are right, in description `dTau` should be just `Tau`